src/.env.sample

```diff
@@ -166,4 +166,14 @@ RESOURCE_AUTO_SAVE_TIMER="30000"
 MAX_RESOURCE_NOTE_LENGTH="256"
 
 # set the instance level default reviewer role for the initialization of review stages
-DEFAULT_REVIEWER_ROLE="reviewer"
\ No newline at end of file
+DEFAULT_REVIEWER_ROLE="reviewer"
+
+# Interface service host address 
+INTERFACE_SERVICE_HOST=http://localhost:3567
+
+# Interface service base URL 
+INTERFACE_BASE_URL=/interface/
+
+# set the end point of interface service to fetch the resources list from the consumption side
+# mandatory only if CONSUMPTION_SERVICE is not self
+BROWSE_EXISTING_END_POINT=v1/internal/fetchResources
```

Comment:
@adithyadinesh0412 this need not be an env config

Reply:
removed from env.